bigbluebutton-html5/imports/api/users-settings/server/methods/addUserSettings.js

```diff
@@ -67,6 +67,10 @@ const currentParameters = [
   'bbb_change_layout',
   'bbb_direct_leave_button',
   'bbb_default_layout',
+  // TRANSCRIPTION
+  'bbb_transcription_partial_utterances',
```

Comment:
TODO: we need to document these in https://github.com/bigbluebutton/bigbluebutton/blob/v2.7.x-release/docs/docs/administration/customize.md?plain=1#L1523